File: output/StarCoder/codenet/C++/Java/s716198307.java
import java.util.*;
public class s716198307 {
    public static void main(String[] args){
        Scanner cin=new Scanner(System.in);
        int n,a[],avg=0,s,ans;
        n=cin.nextInt();
        a=new int[n];
        for(int i=0;i<n;i++){
            a[i]=cin.nextInt();
            avg+=a[i];
        }
        avg/=n;
        s=Math.abs(a[0]-avg);
        ans=0;
        for(int i=1;i<n;i++){
            if(s>Math.abs(a[i]-avg)){
                s=Math.abs(a[i]-avg);
                ans=i;
            }
        }
        System.out.println(ans);
    }
}

File: output/StarCoder/codenet/C++/Python/s716198307.py
import sys

n = int(raw_input().strip())
a = [int(a_temp) for a_temp in raw_input().strip().split(' ')]
avg = sum(a)/n
s = abs(a[0] - avg)
ans = 0
for i in range(1, n):
    if s > abs(a[i] - avg):
        s = abs(a[i] - avg)
        ans = i
print ans